tldraw/tldraw
# refactor(examples): add new examples and clean up existing ones

This PR adds several new examples and cleans up each example. It includes a new "develop" example to be used while developing.

### Change type


### Test plan

1. Open the examples site and verify the new "develop" example works.
2. Verify existing examples still function as expected.


### Release notes

- Added new developer examples and improved existing example code structure.
